packages/dx-react-grid/docs/reference/data-type-provider.md

```diff
@@ -47,6 +47,7 @@ column | [Column](grid.md#column) | A column object.
 row? | any | A row.
 value | any | Specifies the editor value.
 onValueChange | (newValue: any) => void | Handles value changes.
+editingEnabled | boolean | Specifies whether editing a column is enabled.
```

Comment:
Maybe rename this property to `readOnly`? Because this name says about editing only.